### ISSUE TITLE
Maps with string keys don't indent properly

### ISSUE THREAD

**Issue by danielberkompas** [elixir-editors/emacs-elixir#359]
Operating System: El Capitan
Emacs: 24.5.1
elixir-mode: 2.3.1 (20160511.2327)

When I try to type a map with string keys, like so:

``` elixir
  %{
    "data" => %{
      "foo" => %{
        "bar" => nil
      }
   }
```

I get this indentation:

``` elixir
  %{
    "data" => %{
  "foo" => %{
"bar" => nil
}
}
  }
```

However, it works properly when the keys are atoms:

``` elixir
  %{
    data: %{
      foo: %{
        bar: nil
      }
    }
  }
```

**Comment by gausby** [comment]
I've added a failing unit test for this scenario.

**Comment by Trevoke** [comment]
Hopefully this is solved with #433. Please reopen if you still experience the issue.
